yellowbrick/classifier/prcurve.py

```diff
@@ -145,16 +148,18 @@ class PrecisionRecallCurve(ClassificationScoreVisualizer):
     """
 
     def __init__(self, model, ax=None, classes=None, fill_area=True, ap_score=True,
-                 micro=True, iso_f1_curves=False, per_class=False, fill_opacity=0.2,
+                 micro=True, iso_f1_curves=False,f1_values=[0.2,0.4,0.6,0.8], per_class=False, fill_opacity=0.2,
                  line_opacity=0.8, **kwargs):
         super(PrecisionRecallCurve, self).__init__(model, ax=ax, classes=classes, **kwargs)
 
         # Set visual params
+        print("monkey")
```

Comment:
:-) I assume this was left over from debugging to make sure that you got to this point in the code?

Reply:
haha yes. removed it.